tasks/simplifications.json

```diff
@@ -348,7 +348,56 @@
 		"maxmemory": "8192",
 		"skipfailures": "no"
 	},
-	"uk.census.postcodeareas.shorelineclippedpostcodeareas_": {
+	"uk.datashare.shorelineclippedpostcodeareas_": {
+		"simplification": "mapshaper",
```

Comment:
Probably not for now, but maybe something to take into account for the next time. This configuration looks like "safe defaults":

```
			"simplification": "mapshaper",
		"geomfield": "wkb_geometry",
		"factor": "9",
		"maxmemory": "8192",
		"skipfailures": "no"
```

Maybe it could be set as default and stop repeating it over and over. That way, in addition, if safe defaults changed (for example, `maxmemory`), it should only be set once.

Reply:
You are right, and that's something that we discussed when we began with the simplifications. Now that it's getting bigger may be a good moment for a refactor. I've created an issue: https://github.com/CartoDB/bigmetadata/issues/558